containers/foreman.yml

```diff
@@ -27,7 +27,7 @@
       tags:
         - always
     - set_fact:
-        application_hostname: "{{ application_hostname | default('') }}"
+        application_hostname: "{{ application_hostname | default('foreman.' + lookup('pipe', 'minishift ip') + '.nip.io') }}"
```

Comment:
Won't this break things if we're not running on minishift?

Why add this here?

Reply:
Oh good point, I'll expand this to only set this if no application_hostname and minishift is false. That way we still provide a solid default for base minishift use case.

Reply:
Minishift should be giving you a `nip.io` hostname by default though. Is it not?